x-pack/filebeat/module/cisco/asa/test/additional_messages.log-expected.json

```diff
@@ -1908,7 +1919,7 @@
         "cisco.asa.message_id": "106014",
         "cisco.asa.source_interface": "fw111",
         "destination.address": "10.10.10.10(type",
```

Comment:
Seems to be an issue with the output from this field?

Reply:
It is because it is looking in this grok for a destination.adress and this includes "(type" up to the space.
It is not from my commit, but I will see if I can fix it.

Reply:
https://community.cisco.com/t5/network-security/asa-implicit-deny/m-p/2175226
maybe there is an error in the example logs like you said, and it is better to insert a space like in the link between destination address and the brackets.